docs/docs/general/02-Getting Started/write-your-first-automation.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_position: 3
+---
+
+# Write Your First Automation
+
+:::tip
+What you will learn
+
+- Initialize your first automation suite
+- Configure your credentials (workspace id and access token)
+- Understand the generated automation case
+- Run your first automation
+- Troubleshooting any issues
+- Where to go next
+:::
+
+## Quickstart
+
+## Initialization
+
+To create your first automation suite, enter
+
+```shell
+npx askui init
+```
+
+<details>
+  <summary>This will create</summary>
+- a `tsconfig.json`: [a json file specifying the root files and the compiler options required to compile the project](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html),
+- a folder called `test` which includes:
+  - `test/my-first-askui-test-suite.test.ts`: an example test with askui,
+  - a folder `helper` which contains `jest.setup.ts` file: this file is setting up the test environment for all tests
+  - a `jest.config.ts`: file for configuration of Jest
+</details>
```

Comment:
There is no space between bullet points. Can you format this part for readability?